Source/Plugins/NetworkEvents/NetworkEvents.cpp

```diff
@@ -263,14 +264,14 @@ String NetworkEvents::handleSpecialMessages(const String& s)
 
             if (key.compareIgnoreCase("Channel") == 0)
             {
-                //Make sure in range
+                // Make sure in range
                 if (value <= 8 && value >= 0)
                 {
                     channel = value;
                 }
                 else
                 {
-                    //Throw some kind of error here
+                    jassertfalse;
```

Comment:
I wouldn't assert here, since it doesn't indicate a bug in the code, we just have invalid user input. Instead, I would return something like "InvalidChannel" as the response, so the message sender is informed that it sent something invalid.